### PR TITLE
issue #36 is fixed

### LINKED ISSUE #36: Add go to top button
Since the home page will be quite long, a go to top button will be useful. It can be added in the bottom right corner.

### PR DESCRIPTION
**Pull request by yuvraj-aseri**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,6 +17,10 @@
     <link rel="stylesheet" href="./assets/css/main.css">
     <!-- fonts -->
     <link rel="preconnect" href="https://fonts.gstatic.com">
+    <!--scroll up button-->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
+<link href="https://fonts.googleapis.com/css?family=Merriweather:400,900,900i" rel="stylesheet">
+
```
Comment (gajrajgchouhan) [inline index.html]: Since we already have font-awesome (see above) and the font isn't being used these can be removed.

File: index.html [hunk 2]
```diff
@@ -373,6 +377,12 @@ <h3 class="section-title">Photos</h3>
     <script defer src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/js/bootstrap.min.js" integrity="sha384-+YQ4JLhjyBLPDQt//I+STsc9iw4uQqACwlvpslubQzn4u2UU2UFM80nGisd026JF" crossorigin="anonymous"></script>
     <script defer src="https://cdnjs.cloudflare.com/ajax/libs/OwlCarousel2/2.3.4/owl.carousel.min.js"></script>
     <script defer src="assets/js/main.js"></script>
+
+   
+    <a id="button"></a>
+
+    
+
```
Comment (gajrajgchouhan) [inline index.html]: Whitspaces can be removed and can you change the id to something related to its function (for eg `id="go-to-top"`). Also  then move the anchor tag to somewhere near top of the `<body>`

**Review by gajrajgchouhan — CHANGES_REQUESTED** [review]

**Review by gajrajgchouhan — APPROVED** [review]
Nice work @yuvraj-aseri ! Thank you for the PR! :rocket:

**Review by harnaman-hk — CHANGES_REQUESTED** [review]
Following improvements would be good
1. Centre the ^ symbol within the circle
2. Size of circle can be reduced, it is slightly larger than convenient for mobile view
3. Use the darker shade of blue in the background gradient

**Comment by harnaman-hk** [comment]
@yuvraj-aseri  Looks good!
@gajrajgchouhan If you don't have any comments, we can merge it soon.

**Review by harnaman-hk — APPROVED** [review]

**Pull request merged**